tubular/edx_api.py

```diff
@@ -3,18 +3,18 @@
 """
 import logging
 from contextlib import contextmanager
+from urllib.parse import urljoin
 
 import backoff
-from requests.exceptions import ConnectionError
+import requests
+from edx_rest_api_client.auth import SuppliedJwtAuth
+from edx_rest_api_client.client import REQUEST_CONNECT_TIMEOUT, REQUEST_READ_TIMEOUT
+from requests.exceptions import ConnectionError, HTTPError, RequestException
```

Comment:
You'll get the linter error here
add `  # pylint: disable=redefined-builtin` to avoid it

Reply:
in this case the local import did not fail